Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM alpine:3.18 AS building
 MAINTAINER TeskaLabs Ltd (support@teskalabs.com)
+USER root
```

Comment:
Why?

Reply:
for ease of execution and to grant necessary permissions for subsequent commands.
I can get rid of it, if it's a concern.

Reply:
Are there any subsequent commands?

Reply:
@eliska-n  - fyi, I believe that this is unnecessary, discuss this in the team, please.

Reply:
probably because of L34 https://github.com/TeskaLabs/asab-iris/pull/54/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R49

Reply:
The Dockerfile is quite oldschool, we will upgrade it a bit. :)